src/bidiMapper/domains/context/bidiPreloadScript.ts

```diff
@@ -116,6 +116,7 @@ export class BidiPreloadScript {
       'Page.addScriptToEvaluateOnNewDocument',
       {
         source: this.#getEvaluateString(),
+        runImmediately: true,
```

Comment:
it appears that this runs for existing targets too whereas the spec probably says that it should only apply to the future targets (probably requires checking if this runs for a target that is actually waiting for a debugger)

Reply:
Actually, I got it. We run immediately for new targets, but not for pre-existing ones. Will push soon.

Reply:
Fixed